lib/rules/require-meta-docs-suggestion.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const utils = require('../utils');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'require suggestable rules to implement a `meta.docs.suggestion` property',
+      category: 'Rules',
+      recommended: false,
+    },
+    type: 'problem',
+    messages: {
+      shouldBeSuggestable: 'Suggestable rules should specify a `meta.docs.suggestion` property with value `true`.',
+      shouldNotBeSuggestable: 'Non-suggestable rules should not specify a `meta.docs.suggestion` property with value `true`.',
+    },
+    schema: [],
+  },
+
+  create (context) {
+    const sourceCode = context.getSourceCode();
+    const ruleInfo = utils.getRuleInfo(sourceCode.ast);
+    let contextIdentifiers;
+    let ruleReportsSuggestions;
+
+    return {
+      Program (node) {
+        contextIdentifiers = utils.getContextIdentifiers(context, node);
+      },
+      CallExpression (node) {
+        if (
```

Comment:
it could be simple to use the util: https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/a3017e253db08630da3f92428df6f4f2680ecf12/lib/utils.js#L209 and [selectors](https://eslint.org/docs/developer-guide/selectors#selectors). :)

Reply:
Will look into this.

Reply:
I'm not sure if it's convenient to use selectors here since we have a variable `contextIdentifiers` in the condition.